[INTERNAL] Adjust documentation regarding package managers (#797)
SAP/ui5-tooling: docs/pages/SAPUI5.md

## Overview

> SAPUI5 libraries are hosted on the public npm registry at `registry.npmjs.org`. However, you should not install them using node package managers e.g. npm. Instead, please let the UI5 Tooling handle them by following this guide.

Reviewer: ```suggestion
SAPUI5 libraries are hosted on the public npm registry at `registry.npmjs.org`. However, you should not install them using node package managers such as npm. Let UI5 Tooling handle them instead by following this guide.
```
